dev/Repeater/ElementManager.cpp

```diff
@@ -314,7 +314,8 @@ void ElementManager::DataSourceChanged(const winrt::IInspectable& /*source*/, wi
             break;
 
         case winrt::NotifyCollectionChangedAction::Move:
-            throw winrt::hresult_not_implemented();
+            OnItemsRemoved(args.OldStartingIndex(), 1);
```

Comment:
>1 [](start = 52, length = 1)

could we move more than one items ? Perhaps we should support moving n items based on the oldItems size ?

Reply:
Done!